app/src/main/java/com/sschuraytz/charcoaldrawing/UndoRedo.java

```diff
@@ -59,6 +59,16 @@ public void redo() {
         }
     }
 
+    public void resetButtonsForNewCanvas() {
```

Comment:
This method does a little more than reset buttons. Actually one of the advantages of UndoRedo is that it doesn't know what is reacting to it so this method shouldn't have "buttons" in it.